src/cmake/externalpackages.cmake

```diff
@@ -569,5 +568,37 @@ macro (find_or_download_pybind11)
     endif ()
 endmacro()
 
+###########################################################################
+# Tessil/robin-map
+
+set (BUILD_ROBINMAP_VERSION "1e59f1993c7b6eace15032f38b5acb0bc34f530b" CACHE STRING "Preferred Tessil/robin-map version, of downloading/building our own")
```

Comment:
I think it's fine to abbreviate this -- just enough digits to make it very unlikely that it will collide. It's customary to do 6-8 hex digits.